CONTRIBUTING.md

```diff
@@ -52,6 +52,40 @@ If you are maintaining a labwc package for another distro feel free to open an i
 Let's try to stick to sircmpwn's [coding style]. If you're not used to it, you
 can use [checkpatch.pl] to run a few simple formatting checks.
 
+Specific nuances that apply to labwc:
+
+- We prefer `sizeof(*foo)` to `sizeof(struct foo)`
+
+### The use of glib
+
+We try to keep the use of glib pretty minimal for the following reasons:
+
+- The use of glib has been known to make AddressSanitiser diagnose false
+  positives and negatives.
+- Log messages come glib functions look inconsistent.
```

Comment:
Was this meant to say "Log messages coming from glib functions"?

Reply:
Yes. Force-pushed fix.